src/coreclr/.nuget/Microsoft.CrossOsDiag.Private.CoreCLR/Microsoft.CrossOsDiag.Private.CoreCLR.pkgproj

```diff
@@ -4,18 +4,19 @@
     <IsShipping>false</IsShipping>
     <CreatePackedPackage>false</CreatePackedPackage>
     <PackageDescription>Private transport package for .NET Core cross OS diagnostic tooling.</PackageDescription>
+    <CrossDacArtifactsDir>$(ArtifactsBinDir)/coreclr/$(CrossDacOutputDir)</CrossDacArtifactsDir>
```

Comment:
Is this since we now build in the join leg itself? Any reason to not pass it down anymore?